### PR TITLE
feat(catalog-camel): add new connectors

### PR DESCRIPTION
**Pull request by rinaldodev**

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

File: cos-fleet-catalog-connectors/gcp/google-bigquery-0.1/pom.xml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                            </adapter>
+                            <kafka>
+                                <prefix>kafka</prefix>
+                                <name>cos-kafka-source</name>
+                                <version>${cos.kamelets.version}</version>
+                            </kafka>
+                            <dataShape>
+                                <consumes>
+                                    <formats>
+                                        <format>application/octet-stream</format>
```
Comment (lburgazzoli) [inline cos-fleet-catalog-connectors/gcp/google-bigquery-0.1/pom.xml:50]: this need to be confirmed as the related kamelet, declares it consumes json https://github.com/apache/camel-kamelets/blob/main/kamelets/google-bigquery-sink.kamelet.yaml#L64-L65

File: cos-fleet-catalog-connectors/azure/azure-eventhubs-0.1/pom.xml [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+                            </adapter>
+                            <kafka>
+                                <prefix>kafka</prefix>
+                                <name>cos-kafka-sink</name>
+                                <version>${cos.kamelets.version}</version>
+                            </kafka>
+                            <dataShape>
+                                <produces>
+                                    <formats>
+                                        <format>application/octet-stream</format>
```
Comment (lburgazzoli) [inline cos-fleet-catalog-connectors/azure/azure-eventhubs-0.1/pom.xml:50]: the kamelet produces json https://github.com/apache/camel-kamelets/blob/main/kamelets/azure-eventhubs-source.kamelet.yaml#L84-L85

File: cos-fleet-catalog-connectors/saas/salesforce-create-0.1/pom.xml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+                            </adapter>
+                            <kafka>
+                                <prefix>kafka</prefix>
+                                <name>cos-kafka-source</name>
+                                <version>${cos.kamelets.version}</version>
+                            </kafka>
+                            <dataShape>
+                                <consumes>
+                                    <formats>
+                                        <format>application/octet-stream</format>
```
Comment (lburgazzoli) [inline cos-fleet-catalog-connectors/saas/salesforce-create-0.1/pom.xml:54]: json: https://github.com/apache/camel-kamelets/blob/main/kamelets/salesforce-create-sink.kamelet.yaml#L83-L84

File: cos-fleet-catalog-connectors/saas/salesforce-delete-0.1/pom.xml [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+                            </adapter>
+                            <kafka>
+                                <prefix>kafka</prefix>
+                                <name>cos-kafka-source</name>
+                                <version>${cos.kamelets.version}</version>
+                            </kafka>
+                            <dataShape>
+                                <consumes>
+                                    <formats>
+                                        <format>application/octet-stream</format>
```
Comment (lburgazzoli) [inline cos-fleet-catalog-connectors/saas/salesforce-delete-0.1/pom.xml:54]: json https://github.com/apache/camel-kamelets/blob/main/kamelets/salesforce-delete-sink.kamelet.yaml#L79

File: cos-fleet-catalog-connectors/saas/salesforce-update-0.1/pom.xml [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+                            </adapter>
+                            <kafka>
+                                <prefix>kafka</prefix>
+                                <name>cos-kafka-source</name>
+                                <version>${cos.kamelets.version}</version>
+                            </kafka>
+                            <dataShape>
+                                <consumes>
+                                    <formats>
+                                        <format>application/octet-stream</format>
```
Comment (lburgazzoli) [inline cos-fleet-catalog-connectors/saas/salesforce-update-0.1/pom.xml:54]: json https://github.com/apache/camel-kamelets/blob/main/kamelets/salesforce-update-sink.kamelet.yaml#L91

**Comment by lburgazzoli** [comment]
@rinaldodev the PR looks good but the data shapes need to be revisited. In some case it may be enough to remove the explicit dataShape configuration and the plugin should take care to configure it according to the kamelet.

**Comment by lburgazzoli** [comment]
@rinaldodev you need to rebase from master, the catalog files have been moved around

**Comment by rinaldodev** [comment]
@lburgazzoli rebased, removed datashapes. Data shape in openapi was kept since it wouldn't generate without it.

**Review by lburgazzoli — APPROVED** [review]

**Pull request merged**